nnotepad/css/index.css

```diff
@@ -0,0 +1,416 @@
+/* poppins-200 - latin */
+@font-face {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 200;
+  src: url("../common/css/fonts/poppins/poppins-v15-latin-200.eot"); /* IE9 Compat Modes */
```

Comment:
These references to `../common` all need to be `../../common` to reach up out of `nnotepad/css`

Reply:
Thanks, fixed the silly mistake.